magento/magento2
# [GraphQl] Get product reviews will filter by store Id

### Description (*)

Get product reviews will filter by Store Id

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#30337

### Manual testing scenarios (*)

1. Query get product reviews by push header `Store=store_code`
2. Try this query with each storeview then see the result.

### Questions or comments


### Contribution checklist (*)
